src/process_util.rs

```diff
@@ -47,16 +47,33 @@ pub fn squash_stdio(cmd: &mut Command) -> Result<(), String> {
     Ok(())
 }
 
-pub fn capture_stdout(mut cmd: Command) -> Result<Vec<u8>, String> {
+pub struct CapturedOutput {
+    pub stdout: Vec<u8>,
+    pub stderr: Vec<u8>,
+}
+
+impl CapturedOutput {
+    fn new() -> Self {
+        Self {
+            stdout: Vec::with_capacity(1024),
+            stderr: Vec::with_capacity(1024),
+        }
+    }
+}
+
+pub fn capture_output(mut cmd: Command) -> Result<CapturedOutput, String> {
     cmd.stdout(Stdio::piped());
+    cmd.stderr(Stdio::piped());
```

Comment:
I'm not sure this is a good idea to use for every thing that was previously `capture_stdout`. The idea was that errors are visible. Here it will swallow the stderr output and only show error code.